packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.constants.js

```diff
@@ -0,0 +1,11 @@
+export const CUSTOM_SNAPSHOT_NAME_PREFIX = 'snap';
+export const CUSTOM_SNAPSHOT_NAME_SEPARATOR = '-';
+export const STATE_NAME = 'nasha.dashboard.partition.accesses';
```

Comment:
Not too fond of storing  ui router related states in constants like this. what's the need ?

Reply:
:+1: 
This has already been done here : https://github.com/ovh/manager/pull/7495/files#diff-4ba66823bb79f831097d103cc4a5bdd67e8fd97c9aa698c60963c3ae4373effb